asv_bench/benchmarks/inference.py

```diff
@@ -1,117 +1,114 @@
-from .pandas_vb_common import *
-import pandas as pd
+import numpy as np
+import pandas.util.testing as tm
+import pandas._libs.lib as lib
```

Comment:
so this is not backward compat before 0.20., but I think ok for now.

Reply:
Do you recall what this import would be pre 0.20?

Reply:
``from pandas import lib`` (this is in fact what ``pandas/lib.py`` does now, but that will get blown away in 0.22

Reply:
You can import lib from `pandas_vb_common.py`, the back compat is handled there